From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Cline <jcline@redhat.com>
Date: Tue, 17 Sep 2019 17:11:03 +0000
Subject: [PATCH] Pull the RHEL version defines out of the Makefile

This is done via the kernel-put-RHEL-info-into-generated-headers.patch
in the rhpatches branch.

Signed-off-by: Jeremy Cline <jcline@redhat.com>
---
 Makefile | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/Makefile b/Makefile
index 48a7abae02d2..2d826a14824a 100644
--- a/Makefile
+++ b/Makefile
@@ -16,6 +16,7 @@ PHONY := _all
 _all:

 # Set RHEL variables
+# Use this spot to avoid future merge conflicts
 include Makefile.rhelver

 # We are using a recursive build, so we need to do a little thinking
@@ -1172,13 +1173,7 @@ endef
 define filechk_version.h
 	echo \#define LINUX_VERSION_CODE $(shell                         \
 	expr $(VERSION) \* 65536 + 0$(PATCHLEVEL) \* 256 + 0$(SUBLEVEL)); \
-	echo '#define KERNEL_VERSION(a,b,c) (((a) << 16) + ((b) << 8) + (c))'; \
-	echo '#define RHEL_MAJOR $(RHEL_MAJOR)'; \
-	echo '#define RHEL_MINOR $(RHEL_MINOR)'; \
-	echo '#define RHEL_RELEASE_VERSION(a,b) (((a) << 8) + (b))'; \
-	echo '#define RHEL_RELEASE_CODE \
-	$(shell expr $(RHEL_MAJOR) \* 256 + $(RHEL_MINOR))'; \
-	echo '#define RHEL_RELEASE "$(RHEL_RELEASE)"'
+	echo '#define KERNEL_VERSION(a,b,c) (((a) << 16) + ((b) << 8) + (c))'
 endef

 $(version_h): FORCE
-- 
2.26.2

